nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -266,3 +266,5 @@ In addition to numerous new and upgraded packages, this release has the followin
 - The option `services.nomad.extraSettingsPlugins` has been fixed to allow more than one plugin in the path.
 
 - The option `services.prometheus.exporters.pihole.interval` does not exist anymore and has been removed.
+
+- `nextcloud` has an option to enable SSE-C in S3.
```

Comment:
Please see the comment right under the section heading

Reply:
Done